Client/Frontend/TabBar/SearchViewController.swift

```diff
@@ -151,7 +151,8 @@ extension SearchViewController: UITableViewDelegate {
             let suggestion = searchSuggestions[indexPath.row]
 
             // Assume that only the default search engine can provide search suggestions.
-            url = searchEngines?.defaultEngine.searchURLForQuery(suggestion)
+//            url = searchEngines?.defaultEngine.searchURLForQuery(suggestion)
+            url = URIFixup().getURL(searchSuggestions[indexPath.row])
```

Comment:
It was a quick fix, I am sorry for my careless.
I thought there would always be a url since ``indexPath.row < searchSuggestions.count``.
I didn't realize suggestions might not be URLs.